### PR TITLE
feat(inputs.redis_sentinel): Allow username and password

### PR DESCRIPTION
**Pull request by powersj**
fixes: #13836

### CODE REVIEW

File: plugins/inputs/redis/redis.go [hunk 1]
```diff
@@ -26,16 +26,17 @@ import (
...
 
 type Redis struct {
-	Commands []*RedisCommand
-	Servers  []string
-	Username string
-	Password string
+	Commands []*RedisCommand `toml:"commands"`
+	Servers  []string        `toml:"servers"`
+	Username string          `toml:"username"`
+	Password string          `toml:"passowrd"`
```
Comment (srebhan) [inline plugins/inputs/redis/redis.go:38]: Does it make sense to convert this to a `config.Secret` at the same time?

**Review by srebhan — APPROVED** [review]
Just an optional question. Feel free to merge if you answer it with "no". :-)

File: plugins/inputs/redis/redis.go (hunk 1 above)
Comment (srebhan) [reply plugins/inputs/redis/redis.go:38]: Merging this as-is. Feel free to submit a follow-up PR.

Comment (campbelljlowman) [inline plugins/inputs/redis/redis.go:38]: Is `toml:"passowrd"` misspelled?

Comment (powersj) [reply plugins/inputs/redis/redis.go:38]: @campbelljlowman ouch, yes please file an issue please

**Pull request merged**